package.json

```diff
@@ -2,6 +2,7 @@
   "description": "The package.json for unit tests.",
   "devDependencies": {
     "@types/jest": "^26.0.22",
+    "@types/node": "^16.4.12",
```

Comment:
Is there a reason for this change?

Reply:
This wasn't a manual change. I'd assumed this was pulled in when updating node.js. Might be mistaken!

Reply:
Ok, this one should be ok but I would try and undo the changes that were made to the 2 package-lock.json files. As far as I'm aware when something gets merged to master an **npm install** gets run before the vsix extension file is made so the package-lock.json files are updated automatically from the package.json